### PR TITLE
docs: [FC-0074] add follow-up questions to decide which type of hook to use

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds a few follow-up questions to decide whether to use an event or a filter. This is only a proposal to clarify even more which hook to use, so if you think this is redundant, then I'm okay with closing this PR.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mariajgrimaldi!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.


#### :radio_button: Update the status of your PR

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.




#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

File: source/developers/concepts/hooks_extension_framework.rst
Comment (mariajgrimaldi) [inline source/developers/concepts/hooks_extension_framework.rst:157]: If we go with closing this PR, I think we should include at least these last few lines :)

File: source/developers/concepts/hooks_extension_framework.rst
Comment (sarina) [inline source/developers/concepts/hooks_extension_framework.rst:155]: Should this refer to the events doc?

File: source/developers/concepts/hooks_extension_framework.rst
Comment (sarina) [inline source/developers/concepts/hooks_extension_framework.rst:148]: I might say
```suggestion
- If not, maybe because your use case should be the default offering of the platform, contribute changes if they benefit the community, consider submitting a `Product Proposal`_ to begin discussing changes to the default offering.
```

and then link to https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review

**Review by sarina — APPROVED** [review]
A few suggestions but this looks like a good addition!

File: source/developers/concepts/hooks_extension_framework.rst
Comment (mariajgrimaldi) [reply source/developers/concepts/hooks_extension_framework.rst]: Oh - yes!

File: source/developers/concepts/hooks_extension_framework.rst
Comment (mariajgrimaldi) [reply source/developers/concepts/hooks_extension_framework.rst]: Thank you! https://github.com/openedx/docs.openedx.org/pull/618/commits/b253115eb7b78c76a9664d100c5e6ec06372b18b

**Review by sarina — APPROVED** [review]

**Pull request merged**
